monadic-html/src/main/scala/scala/xml/xml.scala

```diff
@@ -146,20 +147,23 @@ The following types are supported:
 - String
 - Boolean (false → remove attribute, true → empty attribute)
 - () => Unit, T => Unit event handler. Note: The return type needs to be Unit!
+- () => Rx[Unit], T => Rx[Unit] event handler, similar to the above.
 - mhtml.Var[T], mhtml.Rx[T] where T is XmlAttributeEmbeddable
 - Option[T] where T is XmlAttributeEmbeddable (None → remove from the DOM)
 """)
 trait XmlAttributeEmbeddable[T]
 object XmlAttributeEmbeddable {
   type XA[T] = XmlAttributeEmbeddable[T]
-  @inline implicit def noneAttributeEmbeddable:                             XA[None.type]    = null
-  @inline implicit def booleanAttributeEmbeddable:                          XA[Boolean]      = null
-  @inline implicit def stringAttributeEmbeddable:                           XA[String]       = null
-  @inline implicit def textNodeAttributeEmbeddable:                         XA[Text]         = null
-  @inline implicit def function0AttributeEmbeddable:                        XA[() => Unit]   = null
-  @inline implicit def function1AttributeEmbeddable[T]:                     XA[T => Unit]    = null
-  @inline implicit def optionAttributeEmbeddable[C[x] <: Option[x], T: XA]: XA[C[T]]         = null
-  @inline implicit def rxAttributeEmbeddable[C[x] <: mhtml.Rx[x], T: XA]:   XA[C[T]]         = null
+  @inline implicit def noneAttributeEmbeddable:                             XA[None.type]      = null
+  @inline implicit def booleanAttributeEmbeddable:                          XA[Boolean]        = null
+  @inline implicit def stringAttributeEmbeddable:                           XA[String]         = null
+  @inline implicit def textNodeAttributeEmbeddable:                         XA[Text]           = null
+  @inline implicit def function0AttributeEmbeddable:                        XA[() => Unit]     = null
+  @inline implicit def function1AttributeEmbeddable[T]:                     XA[T => Unit]      = null
+  @inline implicit def function0AttributeEmbeddableRx:                      XA[() => Rx[Unit]] = null
```

Comment:
Probably need to be `[C[x] <: mhtml.Rx[x]: XA[() => C[Unit]]` like the other one, or scalac won't consider that implicit for `() => Var[Unit]`.